Final Project Submission/Candy_land.cpp

```diff
@@ -0,0 +1,147 @@
+//Program Name: Candy Land
+/*Purpose:  Two player game of candy land first one to 40 wins!*/
+//Author: Justin Hunter
+//Date Last Modified: April 25, 2019
+#include <iostream>
+using namespace std;
+
+string start_game;
+int p1_position;
+int p2_position;
+int roll;
+int current_player = 1;
+int cardnumber;
+
+
+//rolls dice function
+void rolling_dice(){
```

Comment:
rolling the dice calculates a value and puts it into the global `roll` variable.  But the result is a simple integer.  It would have been better to make this an `int` function and _return_ roll rather than setting it as a global.